public/javascripts/register.js

```diff
@@ -1,6 +1,11 @@
 document.addEventListener('DOMContentLoaded', () => {
   const submitButton = document.querySelector('#submit');
 
-  //submitButton.setAttribute('disabled', true);
+  // submitButton.setAttribute('disabled', true);
   // Select required input fields
+
+  const logoContainer = document.querySelector('.logo-container');
+  logoContainer.addEventListener('click', () => {
+    window.location.href = '/';
+  });
```

Comment:
Can you please explain the rational for having this client side JS handing the redirection? Ideally after the user submits the server would redirect them the to index page.

Reply:
Oh, this was meant so that the user could click on the logo and be redirected to the landing page for the site. The form is on the register page is done with the server redirection.

Reply:
okay in that case please use an anchor tag to link the logo to the landing page of the site.

Reply:
It is resolved in the latest commit to this branch